html/contact.html

```diff
@@ -47,6 +49,14 @@ <h1>Contact Information</h1>
       <p>Ayodele Isaac</p>
     </footer>
   <script src="../js/nav.js"></script>
+  <script src="luxon.min.js"></script>
+  <script>
+    //Trying luxon as a script here
+    let DateTime = luxon.DateTime;
+    let now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
+    const showDate = document.querySelector('.date');
+    showDate.innerHTML = now; 
+  </script>
```

Comment:
- [ ] [OPTIONAL] I'd highly advise you to not mix up your javascript code with the HTML as this will make your code unmanageable and goes against best practices (keep the interface and the logic separate) ✔